.github/workflows/deploy.yml

```diff
@@ -0,0 +1,22 @@
+name: SSH-Deploy
+
+on: 
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: ssh-deply for chat-gui
```

Comment:
nitpick:
```suggestion
    - name: ssh-deploy for chat-gui
```

Reply:
Not a nitpick at all my dude, thanks for the catch.